dinghy-lib/src/config.rs

```diff
@@ -140,6 +140,7 @@ pub struct SshDeviceConfiguration {
     pub username: String,
     pub port: Option<u16>,
     pub path: Option<String>,
+    pub rsync: Option<String>,
```

Comment:
I think we should call it something more specific... we probably want to keep the single word "rsync" for a simpler use case (like pointing to the remote rsync path, maybe).
Maybe something like install_adhoc_rsync_local_path ?